RABBITMQ_INTEGRATION.md

```
@@ -0,0 +1,140 @@
+```yaml
+# config/routing.yaml
+create_message:
+  exchange: "notifications" # Topic exchange for notifications
+  routing_key: "message.created" # Routing key
+```
+
+## Flow
+
+1. **Message Creation**
+
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)